src/FakeItEasy/A.cs

```diff
@@ -14,14 +14,14 @@ namespace FakeItEasy
     [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "A", Justification = "Is spelled correctly.")]
     public static class A
     {
-        private static IFakeConfigurationManager ConfigurationManager
+        internal static IFakeCreatorFacade FakeCreator

```

Comment:
> It could, but then I didn't want knowledge of how the service locator works (are they singletons or not?) to have to propagate to the new type. I thought it safest just to reuse.

Works for me

> Incidentally, I wonder if it would probably cleaner to refactor A to forward calls to Sdk.Create? That would remove the need for the generic members on IFakeCreatorFacade.

I'm not sure I like it... it's yet another indirection level, in a library that already has so many.
Anyway, if you feel it's worth it, don't let me stop you :wink: